modin/engines/ray/cudf_on_ray/frame/data.py

```diff
@@ -15,17 +15,17 @@
 import ray
 
 from .partition import cuDFOnRayFramePartition
-from .partition_manager import cuDFOnRayFrameManager
+from .partition_manager import cuDFOnRayFramePartitionManager
 
 from modin.engines.ray.pandas_on_ray.frame.data import PandasOnRayFrame
 from modin.error_message import ErrorMessage
 
 
 class cuDFOnRayFrame(PandasOnRayFrame):
 
-    _frame_mgr_cls = cuDFOnRayFrameManager
+    _partition_mgr_cls = cuDFOnRayFramePartitionManager
 
-    def _apply_index_objs(self, axis=None):
+    def synchronize_labels(self, axis=None):
         """Eagerly applies the index object (Index or Columns) to the partitions.
```

Comment:
this summary probably needs updating, too

Reply:
Updated